main.go

```diff
@@ -6,18 +6,24 @@ import (
 )
 
 func main() {
-	router := gin.Default()
-	var corsConfig = cors.DefaultConfig()
-	// TODO: for the time being, we allow requests from all origins.
+    router := gin.Default()
+    var corsConfig = cors.DefaultConfig()
+    // TODO: for the time being, we allow requests from all origins.
 	// We can consider narrowing this if/when the Project Ricotta front-end has
 	// specific known origin domains that we want to restrict requests to originating from.
-	corsConfig.AllowAllOrigins = true
-	corsConfig.AllowHeaders = append(corsConfig.AllowHeaders, "authorization")
-	router.Use(cors.New(corsConfig))
-	router.POST("/login", postUserAuthorization)
-	router.GET("/profile", getCurrentUserProfile)
-	router.GET("/profile/:userID", getUserProfileByID)
-	router.POST("/profile", postUserProfile)
+    corsConfig.AllowAllOrigins = true
+    corsConfig.AllowHeaders = append(corsConfig.AllowHeaders, "authorization")
+    router.Use(cors.New(corsConfig))
+    
+    router.POST("/login", postUserAuthorization)
+    router.GET("/profile", getCurrentUserProfile)
+    router.GET("/profile/:userID", getUserProfileByID)
+    router.POST("/profile", postUserProfile)
+    router.POST("/register", postUserRegistration)
+    
+    router.GET("/test", func(c *gin.Context) {
```

Comment:
I'm curious about the purpose of this API. Is this simply something to ping? We could just ping /login with no data and look for the expected 400 BAD REQUEST if we want to see if the system is up.

Reply:
I was going to make a note to remove `router.GET("/test", func(c *gin.Context)` this line of code but was going to leave open to discussion. I think testing can be done separately.